src/tools/pmreorder/consistencycheckwrap.py

```diff
@@ -1,10 +1,9 @@
 # SPDX-License-Identifier: BSD-3-Clause
-# Copyright 2018, Intel Corporation
+# Copyright 2018-2021, Intel Corporation
```

Comment:
This should be a separate PR, or at least a separate commit.

Reply:
👍 +1